src/main/groovy/com/bmuschko/gradle/docker/internal/RegistryAuthLocator.groovy

```diff
@@ -281,7 +281,7 @@ class RegistryAuthLocator {
      * @param image the name of the docker image
      * @return docker registry name
      */
-    private static String getRegistry(String image) {
+    static String getRegistry(String image) {
```

Comment:
This should become a non-static method. We already have the instance of `RegistryAuthLocator` in `DockerPushImage`.